bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import os
 from functools import partial
+from typing import Iterable
```

Comment:
Unfortunately we cannot use `typing` in the facade, we still support Python 2.7 here :sweat: